app/forms/sections/plug-ins-or-extensions.json

```diff
@@ -0,0 +1,40 @@
+{
+  "id": "plug-ins-or-extensions",
+  "title": "Browser based - plug-ins or extensions",
+  "mainAdvice": "Answer the following questions about your browser based Solution",
+  "additionalAdvice": [
+    "This information will be visible to buyers on your Solution page.",
+    "* indicates required field."
+  ],
+  "questions": {
+    "plugins-required": {   
+      "mainAdvice": "Are any plug-ins or browser extensions required?*",
+      "additionalAdvice": "Please select below if your Solution requires any plug-ins or browser extentions.",
+      "type": "radiobutton-options",
+      "errorResponse": {
+        "required": "Please select whether or not plug-ins or browser extensions are required"
+      },
+      "options": [
```

Comment:
Take this guy out. We shouldn't need it anymore

Reply:
Correct. I'll remove this.